### DISCUSSION TITLE
纯文本也使用CQ码

### DISCUSSION THREAD

**Discussion by kanrichan** [howmanybots/onebot#97] [想法 / 建议]
比如[CQ:text,text=hello]
- 有效防止注入问题
- 降低序列化与反序列化难度
- 统一CQ码的转义问题

**Comment by kanrichan** [comment]
细思了下，好像没有必要，此贴终结

**Comment by ExerciseBook** [comment]
其实确实可以，但是不作强求？因为 array 格式的上报负责就是这样的。

**Comment by stdrc** [comment]
我觉得可以把 cq 码构成的 string 作为 array 的对称表示，纯文本确实可以表示成 text 消息段，然后新增一个字段把每个 segment 的 alt text 合并成一个纯文本

**Reply by stdrc** [reply]
这样如果一个应用只看纯文本，那同时他也能看到富媒体消息的 alt text，听起来不错，如果他要具体地解析，他可以选择 string 或 array 类型，看喜好

**Reply by stdrc** [reply]
关联 https://github.com/howmanybots/onebot/discussions/72

**Comment by shigma** [comment]
或许可以 `[CQ:text,content=]`？两边 text 会不会有点语义重复的感觉？
